demo/src/screens/componentScreens/CarouselScreen.js

```diff
@@ -1,70 +1,98 @@
 import _ from 'lodash';
```

Comment:
There's a separate example screen for the PageControl. 
I think it worth update it a little to show the new capabilities (maybe something simple with Next and Back buttons)
(another open is to use the Carousel as an example screen for the PageControl)

Reply:
Added